content/movie-review-pt1.md

```diff
@@ -0,0 +1,464 @@
+# Create a Basic Program: Part 1
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Use basic Rust language features for encapsulating code
+- Deserialize instruction data into Rust data types
+- Execute different program logic for different types of instructions
+- Explain the structure of a smart contract on Solana
+
+# TL;DR
```

Comment:
@jamesrp13 for the TL;DR do we want to bold any terms?  maybe **multiple discrete instructions** or **enums**?